arch/risc-v/src/common/supervisor/riscv_sbi.c

```diff
@@ -69,7 +69,9 @@ static inline uint64_t rdtime(void)
 
 void riscv_sbi_set_timer(uint64_t stime_value)
 {
+#if 0
```

Comment:
Just to test out that all the other code passes CI. One option is to omit the knsh file from this patch. Another option is to remove or flag out this error temporarily.